dataviz/ggplot2.qmd

```diff
@@ -200,9 +200,9 @@ murders |> ggplot(aes(population/10^6, total)) +
 
 Clearly, the second call to `geom_text` does not use `population` and `total`.
 
-## Non-aesthetic arguments
+## Static aesthetics
```

Comment:
I hope this is ok -  I do believe the distinction is mapping vs setting, not aesthetic vs non-aesthetic.